volatility/framework/layers/vmware.py

```diff
@@ -48,8 +48,7 @@ def _read_header(self) -> None:
         if magic not in [b"\xD2\xBE\xD2\xBE"]:
```

Comment:
This probably needs updating if version is ever supposed to be different.  At the moment magic must be a fixed value, so version could be pre-calculated...

Reply:
Fixed in next commit